app-emulation/spice-vdagent/spice-vdagent-0.22.1.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit linux-info tmpfiles
+
+DESCRIPTION="SPICE VD Linux Guest Agent"
+HOMEPAGE="https://www.spice-space.org/"
+SRC_URI="https://www.spice-space.org/download/releases/${P}.tar.bz2"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk selinux systemd"
+
+DEPEND="
+	dev-libs/glib:2
+	>=app-emulation/spice-protocol-0.14.0
+	media-libs/alsa-lib
+	sys-apps/dbus
+	x11-libs/libdrm
+	>=x11-libs/libpciaccess-0.10
+	x11-libs/libXfixes
+	x11-libs/libXrandr
+	x11-libs/libX11
+	x11-libs/libXinerama
+	gtk? ( x11-libs/gtk+:3 )
+	systemd? ( sys-apps/systemd )"
+RDEPEND="${DEPEND}
+	selinux? ( sec-policy/selinux-vdagent )"
+BDEPEND="virtual/pkgconfig"
+
+CONFIG_CHECK="~INPUT_UINPUT ~VIRTIO_CONSOLE"
+ERROR_INPUT_UINPUT="User level input support (INPUT_UINPUT) is required"
+ERROR_VIRTIO_CONSOLE="VirtIO console/serial device support (VIRTIO_CONSOLE) is required"
+
+src_configure() {
+	local opt=()
+	if use systemd; then
+		opt+=( --with-session-info=systemd )
+	else
+		opt+=( --with-session-info=none )
+	fi
+
+	econf \
+		--with-init-script=systemd \
+		--localstatedir="${EPREFIX}"/var \
+		$(use_with gtk) \
+		"${opt[@]}"
+}
+
+src_install() {
+	default
+
+	rmdir -p "${ED}"/var/run/spice-vdagentd || die
```

Comment:
Again, this fails.
```
>>> Install app-emulation/spice-vdagent-0.22.1 into /var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image
make -j16 DESTDIR=/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image install 
make[1]: Entering directory '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/work/spice-vdagent-0.22.1'
/bin/mkdir -p /var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image/var/run/spice-vdagentd
 /bin/mkdir -p '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image/usr/bin'
 /bin/mkdir -p '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image/usr/sbin'
 /bin/mkdir -p '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image/usr/share/gdm/greeter/autostart'
 /bin/mkdir -p '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image/usr/share/gdm/autostart/LoginWindow'
 /bin/mkdir -p '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image/usr/share/man/man1'
 /bin/mkdir -p '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image/usr/lib/tmpfiles.d'
 /usr/lib/portage/python3.10/ebuild-helpers/xattr/install -c -m 644 ./data/spice-vdagent.desktop '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image/usr/share/gdm/greeter/autostart'
 /usr/lib/portage/python3.10/ebuild-helpers/xattr/install -c -m 644 ./data/spice-vdagent.desktop '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image/usr/share/gdm/autostart/LoginWindow'
 /bin/mkdir -p '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image/lib/udev/rules.d'
 /usr/lib/portage/python3.10/ebuild-helpers/xattr/install -c -m 644 ./data/tmpfiles.d/spice-vdagentd.conf '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image/usr/lib/tmpfiles.d'
 /bin/mkdir -p '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image/etc/xdg/autostart'
  /usr/lib/portage/python3.10/ebuild-helpers/xattr/install -c src/spice-vdagent '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image/usr/bin'
  /usr/lib/portage/python3.10/ebuild-helpers/xattr/install -c src/spice-vdagentd '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image/usr/sbin'
 /usr/lib/portage/python3.10/ebuild-helpers/xattr/install -c -m 644 data/spice-vdagent.1 data/spice-vdagentd.1 '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image/usr/share/man/man1'
 /usr/lib/portage/python3.10/ebuild-helpers/xattr/install -c -m 644 ./data/70-spice-vdagentd.rules '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image/lib/udev/rules.d'
 /usr/lib/portage/python3.10/ebuild-helpers/xattr/install -c -m 644 ./data/spice-vdagent.desktop '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image/etc/xdg/autostart'
make[1]: Leaving directory '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/work/spice-vdagent-0.22.1'
rmdir: failed to remove directory '/var/tmp/portage/app-emulation/spice-vdagent-0.22.1/image': Directory not empty
 * ERROR: app-emulation/spice-vdagent-0.22.1::gentoo failed (install phase):
 *   (no error message)
 * 
```
Does it work for you? 

```
       -p, --parents
              remove  DIRECTORY and its ancestors; e.g., 'rmdir -p a/b/c' is similar to 'rmdir a/b/c
              a/b a'
```
`-p` is not the flag you want to be using here.

Please do test your ebuild before pushing to make sure it works as you intend it to :)